### DISCUSSION TITLE
Possible Reasons why a merged PR to the main branch won't get deployed to production?

### DISCUSSION THREAD

**Discussion by mithi** [vercel/community#104] [Help] [answered]
I merged a PR to main about 11 hours ago

<img width="1131" alt="Screen Shot 2021-03-12 at 1 29 37 PM" src="https://user-images.githubusercontent.com/1670421/110896678-0bd6d280-8337-11eb-923a-fbca19d847fe.png">

But the last automatic production deployment is still 4 days ago

<img width="1043" alt="Screen Shot 2021-03-12 at 1 32 29 PM" src="https://user-images.githubusercontent.com/1670421/110896942-8869b100-8337-11eb-9f1b-98d4e0ba9b6e.png">

Any possible reason why the merge is not triggering an automatic deployment? 
I don't think I did anything different and Vercel didn't inform me about any limits I might have reached such
that a deployment would be blocked.

I've read this but the scenarios don't seem apply to my case...
https://vercel.com/support/articles/why-aren-t-commits-triggering-deployments-on-vercel

Thanks!

**Comment by calil-amaral-dynamox** [comment]
I have the same issue here, and I still do not understand what is happening.

**Reply by sara-wvs** [reply]
https://www.vercel-status.com/

> Investigating - We are currently investigating elevated error reports of deployment creation due to an upstream provider outage. Existing deployments and live traffic are not affected by this issue. We will share updates as they become available.
Jan 4, 15:16 UTC

I'm in the same boat. I think it might be this outage?

**Comment by Pieparker** [comment, marked as answer]
If this error persists, given that the OP is several months old, please feel free to check out our documentation or create your own discussion via our [Vercel Community Discussions](https://github.com/vercel/vercel/discussions)!
